src/components/checkbox/index.tsx

```diff
@@ -7,7 +7,7 @@ export interface CheckboxProps {
   isDisabled?: boolean;
   isChecked?: boolean;
   onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
-  label?: string;
+  label?: ReactNode | string;
```

Comment:
Is there reason to have `string` besides `ReactNode`

Reply:
didn't wanted to break contract since the component is already in use